### PR TITLE
MuhammedKalkan.OpenLens version 6.0.1

### PR DESCRIPTION
**Pull request by brotich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78765)

### CODE REVIEW

File: manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+PackageIdentifier: MuhammedKalkan.OpenLens
+PackageVersion: 6.0.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 6.0.1-latest.1660157736010
```
Comment (mdanish-kh) [inline manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml:13]:
```suggestion
- DisplayVersion: 6.0.1-latest.1660667913150
```
This should match control panel.

**Comment by brotich** [comment]
will check on this tomorrow

File: manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 6.0.1-latest.1660667913150
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml:15]:
```suggestion
- Architecture: x64
```

File: manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.0.1/OpenLens-6.0.1.exe
+  InstallerSha256: 21228E88B8E1FA18F2A77CCD19E627C1238B79D3274C629E924BC2A3E218139D
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml:22]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by brotich** [comment]
closing for now, till i figure out what the issue is

**Pull request closed without merging**